[2024-02-14T09:07:53.156+0000] {processor.py:161} INFO - Started process (PID=4673) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:07:53.157+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:07:53.157+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:07:53.157+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:07:53.160+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:07:53.159+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:07:53.160+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:07:53.160+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:07:53.160+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:07:53.168+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:08:42.458+0000] {processor.py:161} INFO - Started process (PID=5250) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:08:42.459+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:08:42.460+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:08:42.459+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:08:42.462+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:08:42.462+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:08:42.462+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:08:42.462+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:08:42.463+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:08:42.471+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:09:37.090+0000] {processor.py:161} INFO - Started process (PID=6077) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:09:37.090+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:09:37.091+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:09:37.091+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:09:37.094+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:09:37.093+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:09:37.094+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:09:37.094+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:09:37.094+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:09:37.101+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:10:22.545+0000] {processor.py:161} INFO - Started process (PID=6672) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:10:22.545+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:10:22.546+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:10:22.546+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:10:22.549+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:10:22.548+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:10:22.549+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:10:22.549+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:10:22.549+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:10:22.556+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:11:16.218+0000] {processor.py:161} INFO - Started process (PID=7291) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:11:16.219+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:11:16.220+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:11:16.219+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:11:16.223+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:11:16.223+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:11:16.223+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:11:16.223+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:11:16.224+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:11:16.268+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2024-02-14T09:12:12.832+0000] {processor.py:161} INFO - Started process (PID=7973) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:12:12.833+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:12:12.834+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:12:12.834+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:12:12.837+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:12:12.836+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:12:12.837+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:12:12.837+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:12:12.837+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:12:12.846+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-02-14T09:13:06.950+0000] {processor.py:161} INFO - Started process (PID=8442) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:06.951+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:13:06.951+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:06.951+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:06.954+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:06.954+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:13:06.954+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:06.954+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:13:06.954+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:06.962+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:13:58.444+0000] {processor.py:161} INFO - Started process (PID=8874) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:58.445+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:13:58.445+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:58.445+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:58.448+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:58.447+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:13:58.448+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:13:58.448+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:13:58.448+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:13:58.456+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:14:38.946+0000] {processor.py:161} INFO - Started process (PID=9155) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:14:38.947+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:14:38.947+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:14:38.947+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:14:38.950+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:14:38.950+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:14:38.950+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:14:38.950+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:14:38.950+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:14:38.958+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:15:30.800+0000] {processor.py:161} INFO - Started process (PID=9533) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:15:30.801+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:15:30.801+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:15:30.801+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:15:30.804+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:15:30.804+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:15:30.804+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:15:30.804+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:15:30.805+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:15:30.812+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:16:22.655+0000] {processor.py:161} INFO - Started process (PID=9896) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:16:22.656+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:16:22.656+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:16:22.656+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:16:22.659+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:16:22.659+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:16:22.659+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:16:22.659+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:16:22.659+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:16:22.667+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:17:16.956+0000] {processor.py:161} INFO - Started process (PID=10277) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:17:16.956+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:17:16.957+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:17:16.957+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:17:16.960+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:17:16.959+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:17:16.960+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:17:16.960+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:17:16.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:17:16.968+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:18:06.448+0000] {processor.py:161} INFO - Started process (PID=10668) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:06.449+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:18:06.450+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:06.450+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:06.453+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:06.453+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:18:06.454+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:06.454+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:18:06.454+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:06.462+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-02-14T09:18:59.550+0000] {processor.py:161} INFO - Started process (PID=11051) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:59.551+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:18:59.552+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:59.551+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:59.554+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:59.554+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:18:59.555+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:18:59.555+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:18:59.555+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:18:59.563+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:19:56.194+0000] {processor.py:161} INFO - Started process (PID=11459) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:19:56.195+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:19:56.196+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:19:56.195+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:19:56.198+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:19:56.198+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:19:56.199+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:19:56.198+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:19:56.199+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:19:56.206+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:20:47.997+0000] {processor.py:161} INFO - Started process (PID=12322) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:20:47.998+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:20:47.999+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:20:47.998+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:20:48.001+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:20:48.001+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:20:48.001+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:20:48.001+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:20:48.002+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:20:48.009+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:21:39.532+0000] {processor.py:161} INFO - Started process (PID=12676) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:21:39.534+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:21:39.535+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:21:39.534+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:21:39.539+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:21:39.538+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:21:39.539+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:21:39.539+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:21:39.539+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:21:39.550+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-02-14T09:22:37.666+0000] {processor.py:161} INFO - Started process (PID=13110) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:22:37.666+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:22:37.667+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:22:37.667+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:22:37.669+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:22:37.669+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:22:37.670+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:22:37.670+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:22:37.670+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:22:37.677+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-02-14T09:23:34.188+0000] {processor.py:161} INFO - Started process (PID=13513) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:23:34.189+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:23:34.190+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:23:34.190+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:23:34.194+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:23:34.194+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:23:34.194+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:23:34.194+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:23:34.194+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:23:34.202+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-02-14T09:24:32.031+0000] {processor.py:161} INFO - Started process (PID=13916) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:24:32.031+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:24:32.032+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:24:32.032+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:24:32.035+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:24:32.034+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:24:32.035+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:24:32.035+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:24:32.035+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:24:32.042+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:25:24.966+0000] {processor.py:161} INFO - Started process (PID=14279) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:25:24.967+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:25:24.968+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:25:24.968+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:25:24.970+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:25:24.970+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:25:24.971+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:25:24.971+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:25:24.971+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:25:24.979+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:26:16.721+0000] {processor.py:161} INFO - Started process (PID=14632) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:26:16.722+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:26:16.722+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:26:16.722+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:26:16.725+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:26:16.725+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:26:16.725+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:26:16.725+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:26:16.725+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:26:16.733+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:27:11.457+0000] {processor.py:161} INFO - Started process (PID=15005) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:27:11.457+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:27:11.458+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:27:11.458+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:27:11.461+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:27:11.461+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:27:11.461+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:27:11.461+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:27:11.461+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:27:11.469+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:28:07.791+0000] {processor.py:161} INFO - Started process (PID=15428) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:07.792+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:28:07.793+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:07.792+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:07.795+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:07.795+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:28:07.795+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:07.795+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:28:07.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:07.803+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:28:58.778+0000] {processor.py:161} INFO - Started process (PID=15935) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:58.779+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:28:58.779+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:58.779+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:58.782+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:58.781+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:28:58.782+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:28:58.782+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:28:58.782+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:28:58.789+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:29:53.395+0000] {processor.py:161} INFO - Started process (PID=16318) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:29:53.396+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:29:53.396+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:29:53.396+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:29:53.399+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:29:53.399+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:29:53.399+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:29:53.399+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:29:53.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:29:53.407+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:30:43.489+0000] {processor.py:161} INFO - Started process (PID=16676) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:30:43.490+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:30:43.491+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:30:43.490+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:30:43.493+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:30:43.493+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:30:43.493+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:30:43.493+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:30:43.494+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:30:43.501+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:31:36.728+0000] {processor.py:161} INFO - Started process (PID=17048) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:31:36.728+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:31:36.729+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:31:36.729+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:31:36.732+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:31:36.731+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:31:36.732+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:31:36.732+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:31:36.732+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:31:36.740+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:32:32.670+0000] {processor.py:161} INFO - Started process (PID=17479) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:32:32.671+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:32:32.671+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:32:32.671+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:32:32.674+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:32:32.674+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:32:32.674+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:32:32.674+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:32:32.675+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:32:32.683+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-14T09:33:23.454+0000] {processor.py:161} INFO - Started process (PID=17832) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:33:23.455+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:33:23.456+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:33:23.455+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:33:23.458+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:33:23.458+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:33:23.458+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:33:23.458+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:33:23.459+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:33:23.466+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-14T09:34:15.223+0000] {processor.py:161} INFO - Started process (PID=18200) to work on /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:34:15.224+0000] {processor.py:830} INFO - Processing file /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-14T09:34:15.224+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:34:15.224+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:34:15.227+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:34:15.226+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-14T09:34:15.227+0000] {logging_mixin.py:188} INFO - [2024-02-14T09:34:15.227+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-14T09:34:15.227+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-14T09:34:15.235+0000] {processor.py:183} INFO - Processing /home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
